### ISSUE TITLE
Mobile (Android) - Infinite Spinning on Sessions Dashboard

### ISSUE THREAD

**Issue by m-amarillas** [coolfiresolutions/core-service-desk#91]
ATTENTION: PLEASE DO NOT INCLUDE ANY PERSONALLY IDENTIFABLE INFORMATION (PII) WITHIN THIS REPORT. Please redact any PII from screenshots, videos or supporting media before attaching them to this report. Thank you.

**Describe the bug**
We are receiving reports from Android mobile users that they are able to move past the blue screen now, but they're now getting stuck on the Sessions Dashboard screen. They can just see the "Welcome Back" message and the spinner. Screenshot attached.

This was reported by 3 ReadySet drivers, all Android phones. 

**Where are you seeing this?**
Android Mobile App

**To Reproduce**
Not sure

**Screenshots**
![2021-05-14](https://user-images.githubusercontent.com/53492670/118336611-a68a9380-b4d7-11eb-8dae-98b3a4346d26.png)

**Smartphone (please complete the following information):**
Samsung SM-G781U (Core v1.8)
&
LGE LM-Q730 (Core v1.8)

**Comment by cfsbhawkins** [comment]
We will need more data, such as a username we can login as to verify their data. Being that its a single customer could be bad data on their profile. Please share this info on Slack.

**Comment by m-amarillas** [comment]
Yeah potentially. I reached out to other drivers at American Expediting and haven't gotten any reports of this yet. They actually have reported that their apps have been reliable after the last update.

Let me see if I can uncover anything further with ReadySet
